### DISCUSSION TITLE
I've installed Pop Shell on an Ubuntu virtual machine but I can't seem to get the system tray configuration menu to show up like it does on Pop!_OS

### DISCUSSION THREAD

**Discussion by rdong8** [pop-os/shell#1039] [Q&A] [answered]
And it doesn't seem to be enabled either as nothing is tiling. Is there somewhere I can turn it on from?

[Video](https://user-images.githubusercontent.com/66289396/123995890-c2b99580-d99c-11eb-95d4-64ab6055c685.mp4)

**Comment by mmstick** [comment, marked as answer]
Check gnome-shell logs at `journalctl -o cat -f /usr/bin/gnome-shell` while restarting the shell after enabling the extension. If the panel icon does not appear, the extension is not enabled.

**Reply by rdong8** [reply]
Forgot to enable it, my bad!
